src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -630,6 +630,13 @@ export function registerTerminalActions() {
 			return accessor.get(ITerminalService).getActiveInstance()?.configure();
 		}
 	});
+	MenuRegistry.appendMenuItem(MenuId.TerminalTabsWidgetContext, {
+		command: {
+			id: TERMINAL_COMMAND_ID.CONFIGURE_ACTIVE,
+			title: localize('workbench.action.terminal.configureAction', "Configure Active Terminal")
```

Comment:
This should just be "Configure" in the context menu. Also this actually does configure the _active_ terminal, not the one that was clicked on.